### PR TITLE
NXBT-3289: remove buggy empty-docker-secret

### PR DESCRIPTION
**Pull request by NourNuxeo**

### CODE REVIEW

File: Jenkinsfile [hunk 1]
```diff
@@ -660,6 +660,10 @@ pipeline {
                 // Previous preview deployment needs to be scaled to 0 to be replaced correctly
                 sh "kubectl -n ${PREVIEW_NAMESPACE} scale deployment nuxeo-preview --replicas=0"
               }
+              sh """kubectl create secret generic kubernetes-docker-cfg \
+                  --namespace=${PREVIEW_NAMESPACE} \
+                  --from-literal=.dockerconfigjson="\$(kubectl --namespace platform get secret kubernetes-docker-cfg -ojsonpath='{.data.\\.dockerconfigjson}' | base64 --decode)" \
+                  --type=kubernetes.io/dockerconfigjson --dry-run -o yaml | kubectl apply -f -"""
```
Comment (ataillefer) [inline Jenkinsfile:666]: why not simply
```
kubectl create secret generic kubernetes-docker-cfg \
                  --namespace=${PREVIEW_NAMESPACE} \
                  --from-literal=.dockerconfigjson="\$(kubectl --namespace platform get secret kubernetes-docker-cfg -ojsonpath='{.data.\\.dockerconfigjson}' | base64 --decode)" \
                  --type=kubernetes.io/dockerconfigjson
```

File: Jenkinsfile [hunk 2]
```diff
@@ -660,6 +660,10 @@ pipeline {
                 // Previous preview deployment needs to be scaled to 0 to be replaced correctly
                 sh "kubectl -n ${PREVIEW_NAMESPACE} scale deployment nuxeo-preview --replicas=0"
               }
+              sh """kubectl create secret generic kubernetes-docker-cfg \
+                  --namespace=${PREVIEW_NAMESPACE} \
+                  --from-literal=.dockerconfigjson="\$(kubectl --namespace platform get secret kubernetes-docker-cfg -ojsonpath='{.data.\\.dockerconfigjson}' | base64 --decode)" \
```
Comment (ataillefer) [inline Jenkinsfile:665]: Could you pleas use a constant for 'platform' as we try to do for kubectl commands.
I'd say, after this [line](https://github.com/nuxeo/nuxeo/blob/fix-NXBT-3289-kubernetes-docker-cfg/Jenkinsfile#L21):
```
kubernetesNamespace = 'platform'
```

File: Jenkinsfile (hunk 1 above)
Comment (NourNuxeo) [reply Jenkinsfile:666]: Not sure it works if I paste some bash code directly in a script directive which takes groovy initially?
I can also try and remove it but I wanted to stay consistent with the all the other kubectl calls accros the same file which are wrapped in an sh""" """" call. 
wdyt ?

File: Jenkinsfile (hunk 2 above)
Comment (NourNuxeo) [reply Jenkinsfile]: Changed the single quotes to double quotes on the whole lines where the var is used

File: Jenkinsfile (hunk 1 above)
Comment (troger) [reply Jenkinsfile:667]: @ataillefer is just simplifying the content of the `sh"""`, not saying you should remove the `sh"""` part :)

File: Jenkinsfile [hunk 3]
```diff
@@ -158,7 +159,7 @@ def buildUnitTestStage(env) {
     stage("Run ${env} unit tests") {
       container("maven-${env}") {
         script {
-          setGitHubBuildStatus("platform/utests/${env}", "Unit tests - ${env} environment", 'PENDING')
+          setGitHubBuildStatus("${kubernetesNamespace}/utests/${env}", "Unit tests - ${env} environment", 'PENDING')
```
Comment (troger) [inline Jenkinsfile:162]: I would say this one is not linked to the Kubernetes namespace, it's just a GitHub status name.

Comment (NourNuxeo) [reply Jenkinsfile]: yes I reverted all those gh status changes

**Pull request merged**